### DISCUSSION TITLE
Failed to resolve action download info. Error: Internal Server Error

### DISCUSSION THREAD

**Discussion by adamjer** [community/community#36625] [Actions]
Workflow

    steps:
    - name: Checkout code
      id: checkout-code
      uses: actions/checkout@v3
      with:
        submodules: recursive
        fetch-depth: 0

    - name: Setup problem matcher
      uses: intel-innersource/frameworks.actions.thirdparty.gcc-problem-matcher@v2

    - name: Build release
      id: build-release
      uses: intel-innersource/frameworks.actions.thirdparty.arch-mingw-github-action@v8
      with:
        command: |
          export VERSION_NAME="${GITHUB_REF##*/}-${GITHUB_SHA##*/}"
          ./package-release.sh ${VERSION_NAME} build --no-package
          echo "VERSION_NAME=${VERSION_NAME}" >> $GITHUB_ENV
    - name: Upload artifacts
      id: upload-artifacts
      uses: actions/upload-artifact@v3
      with:
        name: dxvk-${{ env.VERSION_NAME }}
        path: build/dxvk-${{ env.VERSION_NAME }}
        if-no-files-found: error

Im getting this issue

	Secret source: Actions
	Runner is running behind proxy server 'http://XXXXX.com:912' for all HTTP requests.
	Runner is running behind proxy server 'http://XXXXX.com:912' for all HTTPS requests.
	Prepare workflow directory
	Prepare all required actions
	Getting action download info
	Failed to resolve action download info. Error: Internal Server Error
	
	Retrying in 10.611 seconds
	Failed to resolve action download info. Error: Internal Server Error
	
	Retrying in 28.104 seconds
	Error: Failed to resolve action download info.

I checked and these causes problems
frameworks.actions.thirdparty.gcc-problem-matcher@v2
frameworks.actions.thirdparty.arch-mingw-github-action@v8

Is there possibly some error in syntax?
It worked perfectly fine when I was using public repositories
https://github.com/Joshua-Ashton/gcc-problem-matcher
https://github.com/Joshua-Ashton/arch-mingw-github-action
but had to move them to internal due to organization policy.

New actions are set to internal and accesses set accordingly
![image](https://user-images.githubusercontent.com/26136487/196626818-d246931d-6432-4173-941b-c94bfebb7b34.png)

**Comment by ossanna16** [comment]
Hey there @adamjer and thank you for asking a good question! I'm glad you found our community 🙂
